api/user/controller/create.js

```diff
@@ -0,0 +1,21 @@
+import Joi from 'joi';
+import saveUser from '../service/create.js';
+
+export default function save(req, res) {
+  const userSchema=Joi.object({
```

Comment:
move userSchema constant outside save() function
in such case function save() became more readable